============================= test session starts =============================
platform win32 -- Python 3.6.5, pytest-3.6.2, py-1.5.3, pluggy-0.6.0 -- C:\Users\rafae\Anaconda3\python.exe
cachedir: .pytest_cache
rootdir: C:\Users\rafae\codenation\python-8, inifile:
plugins: remotedata-0.2.1, openfiles-0.3.0, doctestplus-0.1.3, arraydiff-0.2
collecting ... collected 3 items

test_submit.py::TestChallenge4::test_verify_signature PASSED             [ 33%]
test_submit.py::TestChallenge4::test_verify_signature_fail PASSED        [ 66%]
test_submit.py::TestChallenge4::test_create_token PASSED                 [100%]

------ generated xml file: C:\Users\rafae\codenation\python-8\output.xml ------
========================== 3 passed in 0.13 seconds ===========================
